vagrant_opm_user/bootstrap.sh

```diff
@@ -20,9 +20,9 @@ sudo apt-get update -y
 
 # OPM packages
 sudo apt-get install libopm-core1-bin -y
-sudo apt-get install libopm-autodiff1-bin -y
-sudo apt-get install libopm-polymer1-bin -y
+sudo apt-get install libopm-simulators1-bin -y
 sudo apt-get install libopm-upscaling1-bin -y
+sudo apt-get install openmpi-bin
```

Comment:
to make it resilient to possible future changes you could use the mpi-default-bin meta package instead.

-------- Opprinnelig melding --------
Fra: "André R. Brodtkorb" <notifications@github.com>
Dato: 03.05.2016 21:35 (GMT+01:00)
Til: OPM/opm-utilities <opm-utilities@noreply.github.com>
Emne: Re: [OPM/opm-utilities] Fixes for 2016 release (#13)


In vagrant_opm_user/bootstrap.sh<https://github.com/OPM/opm-utilities/pull/13#discussion_r61941233>:

>  sudo apt-get install libopm-upscaling1-bin -y
> +sudo apt-get install openmpi-bin


Without this manually added dependency, I get errors trying to run flow.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/OPM/opm-utilities/pull/13/files/41c3ec52cd8702e5b3f0e72f0a0acb98208378c9#r61941233>